### DISCUSSION TITLE
Issues deploying to vercel.

### DISCUSSION THREAD

**Discussion by dalist1** [DuCanhGH/next-pwa#113] [General]
Hi, I'm having issues deploying to Vercel, started happening recently.

![image](https://github.com/DuCanhGH/next-pwa/assets/52313111/1bc2a5cf-0f39-4f61-85b5-b4ecf431492a)

**Comment by DuCanhGH** [comment]
@dalist1 Can this issue be reproduced with `next-pwa` not installed? Seems like more like a Next.js issue than a `next-pwa` issue, to be honest. That you see `@ducanh2912/next-pwa` in the path is simply due to how your package manager manages your dependencies, most likely.

P/s: sorry for rejecting your friend request on Discord, but you didn't introduce yourself there and I had no way to know who you are :)

**Reply by dalist1** [reply]
@DuCanhGH Can't seem to reproduce this without `@ducanh2912/next-pwa` installed. Seems to be an issue with how next is caching `@ducanh2912/next-pwa`, here's my `next-pwa` config:

```javascript
const withPWA = require("@ducanh2912/next-pwa").default({
  dest: "public",
  cacheOnFrontEndNav: true,
  aggressiveFrontEndNavCaching: true,
});
```

I've tried deleting `.next` folder and reinstalling all the packages.

**Reply by DuCanhGH** [reply]
@dalist1 I'd be really pissed off if they somehow managed to cache `next.config.js` for god knows what, to be honest. Perhaps you can try doing this if it doesn't happen during the build process:
```js
const {
  PHASE_DEVELOPMENT_SERVER,
  PHASE_PRODUCTION_BUILD,
} = require("next/constants");

/** @type {import("next").NextConfig} */
const nextConfig = {
  reactStrictMode: true,
};

module.exports = (phase) => {
  if (phase === PHASE_DEVELOPMENT_SERVER || phase === PHASE_PRODUCTION_BUILD) {
    const withPWA = require("@ducanh2912/next-pwa").default({
      dest: "public",
    });
    return withPWA(nextConfig);
  }
  return nextConfig;
};
```

To be frankly honest, the error message is way too vague for me to know what it is about. I can only point the finger at Next.js doing some weird stuff or your package manager acting strange. Can you try providing me with more information and check whether this issue happens in Next.js versions older than 14.0.0? Seems that the docs site works perfectly (it's on `next@13.5.5`, and it is deployed to Vercel, same as your app).

If you can make a minimal reproduction, please do so. Thanks!
